guides/release/accessibility/page-template-considerations.md

```diff
@@ -15,14 +15,76 @@ Consider this format:
 
 Note that the unique page title is first. This is because it is the most important piece of information from a contextual perspective. Since a user with a screen reader can interrupt the screen reader as they wish, it introduces less fatigue when the unique page title is first, but provides the additional guidance if it is desired. 
 
-Until Ember provides this functionality by default, there are a few different Ember addons that will help:
+One simple way to add page titles is to create a `title` helper:
+
+```javascript {data-filename=app/helpers/title.js}
+import Helper from '@ember/component/helper';
+
+export default class Title extends Helper {
+  original = document.title;
+
+  compute([title]) {
+    document.title = title;
+  }
+
+  willDestroy() {
+    document.title = this.original;
+  }
+}
+
+```
+
+We can use this helper to set the page title at any point in any template.
+
+For example, if we have a “posts” route, we can set the page title for it like so:
+
+
+```handlebars {data-filename=app/routes/posts.hbs}
+{{title "Posts - Site Title"}}
+
+{{outlet}}
+```
+
+Extending the example, if we have a “post” route that lives within the “posts” route, we could set its page title like so:
+
+```handlebars {data-filename=app/routes/posts/post.hbs}
+{{title (concat @model.title " - Site Title")}}
+
+<h1>{{@model.title}}</h1>
+```
+
+This title will take effect when we enter the “post” route, and the line of code in our `willDestroy` hook will take care of restoring the former title when we return to the “posts” route.
+
+This technique is a reasonable first step, but has limitations:
+
+- It does *not* work when the page is rendered server-side with FastBoot.
+- It doesn’t provide any conventions for constructing nested page titles.
```

Comment:
```suggestion
- It doesn't provide any conventions for constructing nested page titles.
```

Not familiar with this linter yet, but seems to be giving a warning https://travis-ci.org/github/ember-learn/guides-source/builds/676236275#L545

I will set up these linters locally after this!